mcache/datadog_checks/mcache/mcache.py

```diff
@@ -120,6 +119,10 @@ def _process_response(cls, response):
         if not len(stats):
             raise BadResponseError("Malformed response for host: {}".format(stats))
 
+        if 'version' in stats:
+            self.set_metadata('version', self.normalize(stats['version']))
```

Comment:
`normalize` method seems to be intended for metrics names only: https://github.com/DataDog/integrations-core/blob/57e8ead12c248904e9da6fd672a80b51f4fd1b07/datadog_checks_base/datadog_checks/base/checks/base.py#L605-L631

Reply:
Should I replace it with .decode('utf-8')?